graph/formats/dot/internal/parser/parser_test.go

```diff
@@ -86,6 +88,38 @@ func TestParseFile(t *testing.T) {
 	}
 }
 
+func TestParseFuzz(t *testing.T) {
+	r, err := zip.OpenReader("../../fuzz/corpus.zip")
+	if err != nil {
+		if os.IsNotExist(err) {
+			t.Skip("no corpus")
```

Comment:
Likewise here, is a `return` statement needed?

Reply:
Same.